core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -49,49 +49,56 @@
   // across all nodes in an instance)
   RPC_PREFIX("rpc.", null, PropertyType.PREFIX,
       "Properties in this category related to the configuration of SSL keys for"
-          + " RPC. See also instance.ssl.enabled"),
+          + " RPC. See also instance.ssl.enabled",
+      "1.6.0"),
   RPC_SSL_KEYSTORE_PATH("rpc.javax.net.ssl.keyStore", "", PropertyType.PATH,
-      "Path of the keystore file for the server's private SSL key"),
+      "Path of the keystore file for the server's private SSL key", "1.6.0"),
   @Sensitive
   RPC_SSL_KEYSTORE_PASSWORD("rpc.javax.net.ssl.keyStorePassword", "", PropertyType.STRING,
       "Password used to encrypt the SSL private keystore. "
-          + "Leave blank to use the Accumulo instance secret"),
+          + "Leave blank to use the Accumulo instance secret",
+      "1.6.0"),
   RPC_SSL_KEYSTORE_TYPE("rpc.javax.net.ssl.keyStoreType", "jks", PropertyType.STRING,
-      "Type of SSL keystore"),
+      "Type of SSL keystore", "1.6.0"),
   RPC_SSL_TRUSTSTORE_PATH("rpc.javax.net.ssl.trustStore", "", PropertyType.PATH,
-      "Path of the truststore file for the root cert"),
+      "Path of the truststore file for the root cert", "1.6.0"),
   @Sensitive
   RPC_SSL_TRUSTSTORE_PASSWORD("rpc.javax.net.ssl.trustStorePassword", "", PropertyType.STRING,
-      "Password used to encrypt the SSL truststore. Leave blank to use no password"),
+      "Password used to encrypt the SSL truststore. Leave blank to use no password", "1.6.0"),
   RPC_SSL_TRUSTSTORE_TYPE("rpc.javax.net.ssl.trustStoreType", "jks", PropertyType.STRING,
-      "Type of SSL truststore"),
+      "Type of SSL truststore", "1.6.0"),
   RPC_USE_JSSE("rpc.useJsse", "false", PropertyType.BOOLEAN,
       "Use JSSE system properties to configure SSL rather than the " + RPC_PREFIX.getKey()
-          + "javax.net.ssl.* Accumulo properties"),
+          + "javax.net.ssl.* Accumulo properties",
+      "1.6.0"),
   RPC_SSL_CIPHER_SUITES("rpc.ssl.cipher.suites", "", PropertyType.STRING,
-      "Comma separated list of cipher suites that can be used by accepted connections"),
+      "Comma separated list of cipher suites that can be used by accepted connections", "1.6.1"),
   RPC_SSL_ENABLED_PROTOCOLS("rpc.ssl.server.enabled.protocols", "TLSv1.2", PropertyType.STRING,
-      "Comma separated list of protocols that can be used to accept connections"),
+      "Comma separated list of protocols that can be used to accept connections", "1.6.2"),
   RPC_SSL_CLIENT_PROTOCOL("rpc.ssl.client.protocol", "TLSv1.2", PropertyType.STRING,
       "The protocol used to connect to a secure server, must be in the list of enabled protocols "
-          + "on the server side (rpc.ssl.server.enabled.protocols)"),
+          + "on the server side (rpc.ssl.server.enabled.protocols)",
+      "1.6.2"),
   /**
    * @since 1.7.0
```

Comment:
This comment is probably unnecessary now with the since info below.

Reply:
I did notice that in a couple places. I left it alone because I did not know if it was important / useful to keep it in the javadoc. I will remove this if it is not needed anymore.

Reply:
Feel free to get rid of them.